_docs/overview/design-getting-started.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Getting Started
+layout: docs
+category: Overview
+usage: >
+  Read this article to learn how to use the Helix design language, which will help you design faster and provide a more consistent user experience.
+status: stable
+resource: true
+last-modified: 2017-10-24
+---
+
+
+## Prerequisites
+
+<div class="hxRow large">
+
+{% column left:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+To take full advantage of Helix resources, you'll need Sketch. [Download Sketch here](http://www.sketchapp.com). Learn more about using Sketch with [these Pablo video lectures](https://www.youtube.com/watch?v=cEplnCnZuDM&list=PLWlUJU11tp4fEXI8deWhBQAHDv9R23WHB).
```

Comment:
Feels like we need a new line for the "Learn more about using Sketch..." sentence.

Reply:
"You can learn more about Sketch by watching these ...
Are there any other resources that we should point them to?

Reply:
Updated to , "You can learn more about Sketch by watching the following [playlist]"

Not sure the reader needed to know of someone on youtube. I'm sure he introduces himself in every video anyways. 😆